packages/ember-data/lib/system/snapshot.js

```diff
@@ -12,7 +12,7 @@ var get = Ember.get;
   @constructor
   @param {DS.Model} internalModel The model to create a snapshot from
 */
-export default function Snapshot(internalModel) {
```

Comment:
Hmm, [Model](https://github.com/emberjs/data/blob/4745dcb6fa00664ddb19bab9f4913695656f7e1c/packages/ember-data/lib/system/model/model.js#L848) and [Store](https://github.com/emberjs/data/blob/4745dcb6fa00664ddb19bab9f4913695656f7e1c/packages/ember-data/lib/system/store.js#L2155) are `export default`ing at the end. But I am happy to change if this is desired ...